File: cfgs/pythoncfgs/CALCULATE_MAXIMUM_VALUE_USING_SIGN_TWO_NUMBERS_STRING.py
Nodes
1;statement;foo_1 = fun_1(foo_2[0]) - 48;8;
2;conditional;foo < foo;9;
3;statement;return foo_1;14;
4;conditional;foo_1[foo_2] == '0' or foo_3[foo_4] == '1' or foo_5 < 2;10;
5;statement;foo_1 *= fun_1(foo_2[foo_3]) - 48;13;
6;statement;foo_1 += fun_1(foo_2[foo_3]) - 48;11;
7;exit;;
Edges
0;2;
1;3;4;
2;7;
3;5;6;
4;2;
5;2;
6;

File: cfgs/java_cfgs_foo/CALCULATE_MAXIMUM_VALUE_USING_SIGN_TWO_NUMBERS_STRING.java
Nodes
1;special;;;
2;statement;int res = fun_1(0) - '0';14;
3;statement;int i = 1;15;
4;conditional;foo_1 < fun_1();16;
5;conditional;((fun_1(foo_2) == '0') || (fun_2(foo_4) == '1')) || (foo_5 < 2);18;
6;statement;res += fun_1(foo_2) - '0';18;
7;statement;res *= fun_1(foo_2) - '0';19;
8;statement;i++;17;
9;statement;return foo_1;21;
10;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;9;
4;6;7;
5;8;
6;8;
7;4;
8;10;
9;
